forest/shared/Cargo.toml

```diff
@@ -41,6 +41,7 @@ fvm_ipld_encoding.workspace      = true
 fvm_shared                       = { workspace = true, default-features = false }
 git-version                      = "0.3"
 hex.workspace                    = true
+isatty                           = "0.1.9"
```

Comment:
the latest seem to be `0.2`, any particular reason behind `0.1`?

Reply:
0.2 is failing build. 1.9 is stable and works